DotVVM.Diagnostics.StatusPage/StatusViewModel.cs

```diff
@@ -56,7 +58,9 @@ public override Task Load()
                     HasParameters = r.ParameterNames.Any(),
                     DefaultValues = r.DefaultValues,
                     RouteName = r.RouteName,
-                    Status = string.IsNullOrWhiteSpace(r.VirtualPath) ? CompilationState.NonCompilable : CompilationState.None
+                    Status = string.IsNullOrWhiteSpace(r.VirtualPath)
+                        ? CompilationState.NonCompilable
+                        : CompilationState.None
```

Comment:
I'd like to see the benchmark that shows that this change makes the code run faster...